Project/PrivateVault App/Code/UI/Gallery/GalleryView.swift

```diff
@@ -25,14 +25,16 @@ struct GalleryView: View {
 	@State var currentAlert: AlertItem?
 	@State var previewSelection: PreviewSelection?
 	@State var itemBeingDeleted: StoredItem?
+	@State private var folder: Folder?
```

Comment:
Items and folders created could have their `folder` property set to this value, this way they would appear on the current location.
Currently, when creating an item or folder, if the user is inside a folder, the new element is added to the root view.

Reply:
makes sense!